data_safe_haven/utility/enums.py

```diff
@@ -1,11 +1,13 @@
-from enum import Enum
+from enum import UNIQUE, Enum, verify
```

Comment:
Should this be `unique`? 

I get this error when trying to run any `dsh` command after running `hatch shell`:

```
(data-safe-haven) deploydsh@0365a536cd8f:/workspaces/data-safe-haven$ dsh
Traceback (most recent call last):
  File "/home/deploydsh/.local/share/hatch/env/virtual/data-safe-haven/Ce8xTVjM/data-safe-haven/bin/dsh", line 5, in <module>
    from data_safe_haven.cli import main
  File "/workspaces/data-safe-haven/data_safe_haven/cli.py", line 8, in <module>
    from data_safe_haven.commands import (
  File "/workspaces/data-safe-haven/data_safe_haven/commands/__init__.py", line 1, in <module>
    from .admin import admin_command_group
  File "/workspaces/data-safe-haven/data_safe_haven/commands/admin.py", line 7, in <module>
    from .admin_add_users import admin_add_users
  File "/workspaces/data-safe-haven/data_safe_haven/commands/admin_add_users.py", line 4, in <module>
    from data_safe_haven.administration.users import UserHandler
  File "/workspaces/data-safe-haven/data_safe_haven/administration/users/__init__.py", line 1, in <module>
    from .user_handler import UserHandler
  File "/workspaces/data-safe-haven/data_safe_haven/administration/users/user_handler.py", line 5, in <module>
    from data_safe_haven.config import Config
  File "/workspaces/data-safe-haven/data_safe_haven/config/__init__.py", line 1, in <module>
    from .config import Config
  File "/workspaces/data-safe-haven/data_safe_haven/config/config.py", line 19, in <module>
    from data_safe_haven.config.context_settings import Context
  File "/workspaces/data-safe-haven/data_safe_haven/config/context_settings.py", line 19, in <module>
    from data_safe_haven.utility import LoggingSingleton, config_dir
  File "/workspaces/data-safe-haven/data_safe_haven/utility/__init__.py", line 2, in <module>
    from .enums import DatabaseSystem, SoftwarePackageCategory
  File "/workspaces/data-safe-haven/data_safe_haven/utility/enums.py", line 1, in <module>
    from enum import UNIQUE, Enum, verify
ImportError: cannot import name 'UNIQUE' from 'enum' (/usr/local/lib/python3.10/enum.py)
```

Reply:
It's new from Python 3.11 https://docs.python.org/3.11/library/enum.html?highlight=unique#enum.EnumCheck